lib/internal/bootstrap/switches/does_own_process_state.js

```diff
@@ -80,6 +80,9 @@ function wrapPosixCredentialSetters(credentials) {
   function wrapIdSetter(type, method) {
     return function(id) {
       validateId(id, 'id');
+      // Coerce -0 into 0, because that is the most reasonable thing
+      // for system calls
+      if (id === -0) id = 0;
       // Result is 0 on success, 1 if credential is unknown.
       const result = method(id);
```

Comment:
I would like to change this to
```suggestion
      const result = method(id + 0);
```
since I don't like to effect this line
https://github.com/nodejs/node/blob/a13500f5037c1eede3a2d0a0db5fc532c05e14a8/lib/internal/bootstrap/switches/does_own_process_state.js#L86

is this ok?

Reply:
@Himself65 , thanks. Changed it according to your suggestion. PTAL.